tests/slow/src/test/scala/tests/feature/Scala3CodeActionLspSuite.scala

```diff
@@ -118,6 +119,111 @@ class Scala3CodeActionLspSuite
        |""".stripMargin
   )
 
+  check(
+    "single-def",
+    """|object Main {
+       |  def method2(i: Int) = ???
+       |  def main = {
+       |    def inner(i : Int) = method2(i + 23 + <<123>>)
```

Comment:
tbh in all these cases I'd expect 123 to be extracted, especially if it's fully highlighted. It's the last argument to `+`, isn't it?

Reply:
I didn't include `Term.ApplyInfix`, since I was trying to guess best what the user might want to extract. But maybe we should do it similar to Intellij and just suggest a couple of different options for each apply argument? :thinking: